jib-core/src/main/java/com/google/cloud/tools/jib/docker/CliDockerClient.java

```diff
@@ -99,21 +99,41 @@ public List<DescriptorDigest> getDiffIds() throws DigestException {
     }
   }
 
-  /** Default path to the docker executable. */
-  public static final Path DEFAULT_DOCKER_CLIENT = Paths.get("docker");
+  /** Default paths to the docker or alternatives executables. */
+  public static final List<Path> DEFAULT_DOCKER_CLIENT = Arrays.asList(
+          Paths.get("docker"), Paths.get("podman")
+  );
 
   /**
    * Checks if Docker is installed on the user's system by running the `docker` command.
    *
    * @return {@code true} if Docker is installed on the user's system and accessible
    */
   public static boolean isDefaultDockerInstalled() {
-    try {
-      new ProcessBuilder(DEFAULT_DOCKER_CLIENT.toString()).start();
-      return true;
-    } catch (IOException ex) {
-      return false;
+    for (Path dockerClient: DEFAULT_DOCKER_CLIENT) {
+      try {
+        new ProcessBuilder(dockerClient.toString()).start();
+        return true;
+      } catch (IOException ignore) {
+        // wait for all docker clients fails
```

Comment:
I think it'd be useful to at least log an error for which client was used, and had failed